README.md

```diff
@@ -3,12 +3,9 @@
 
 The Coinbase Go SDK enables the simple integration of crypto into your app. By calling Coinbase's Platform APIs, the SDK allows you to provision crypto wallets, send crypto into/out of those wallets, track wallet balances, and trade crypto from one asset into another.
```

Comment:
Do we wanna remove this part "send crypto into/out of those wallets, track wallet balances," ?